runtime/src/accounts_db.rs

```diff
@@ -6186,6 +6182,10 @@ impl AccountsDb {
 
 #[cfg(test)]
 impl AccountsDb {
+    pub fn new(paths: Vec<PathBuf>, cluster_type: &ClusterType) -> Self {
+        Self::new_for_tests(paths, cluster_type)
+    }
```

Comment:
Would it make sense to call `Self::new_with_config()` and then just delete the `new_for_tests()` function?

Reply:
Spoke to soon. I see the reason in `runtime/tests/accounts.rs` below. Sorry!